odl/discr/discr_mappings.py

```diff
@@ -317,31 +292,29 @@ def __init__(self, ip_fset, grid, dspace, order='C', linear=False):
             means the first grid axis varies fastest, the last most
             slowly, 'F' vice versa.
         """
+        linear = True if isinstance(ip_fset, FunctionSpace) else False
         FunctionSetMapping.__init__(self, 'extension', ip_fset, grid, dspace,
-                                    order, linear)
+                                    order, linear=linear)
 
-        # TODO: remove this requirement depending on the vectorization
-        # solution
         if not isinstance(ip_fset.domain, IntervalProd):
             raise TypeError('domain {!r} of the function set is not an '
                             '`IntervalProd` instance.'
                             ''.format(ip_fset.domain))
 
-    # TODO: Implement _apply()
-
-    def _call(self, inp):
-        """The raw `call` method for out-of-place evaluation.
+    def _call(self, x):
+        """The raw method for out-of-place evaluation.
```

Comment:
Change this docstring now (out-of-place is now implicit), perhaps more of what it does?